src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,21 @@
+$box-width: 100px;
+$box-height: 100px;
+$box-gap: 1px;
+$calendar-paddings: 10px;
+$box-width-plus-gap: $box-width + $box-gap;
+$calendar-width: 6 * $box-width-plus-gap + $box-width + $calendar-paddings;
+
+$border-color: black;
+$background-color: #eee;
+
+$days-in-month: 31;
+
+$week-days: (
+  "mon": $box-width-plus-gap * 0,
+  "tue": $box-width-plus-gap * 1,
+  "wed": $box-width-plus-gap * 2,
+  "thu": $box-width-plus-gap * 3,
+  "fri": $box-width-plus-gap * 4,
+  "sat": $box-width-plus-gap * 5,
+  "sun": $box-width-plus-gap * 6
```

Comment:
it would be better to calculate these values in the loop, where you set `margin-left`